extensions/common/context.cc

```diff
@@ -391,6 +395,19 @@ void populateTCPRequestInfo(bool outbound, RequestInfo* request_info,
   request_info->request_protocol = kProtocolTCP;
 }
 
+bool populateGRPCInfo(RequestInfo* request_info) {
+  std::string value;
+  if (!getValue({"filter_state", GrpcStatsName}, &value)) {
+    return false;
+  }
+  std::vector<std::string_view> parts = absl::StrSplit(value, ',');
+  if (parts.size() == 2) {
```

Comment:
Would be great to add a comment here about what are the two parts instead of understanding it from the code. Also is this documented somewhere?

Reply:
It's implemented by grpc_stats and should probably be documented there. I'll do that separately.